vita3k/codec/src/aac.cpp

```diff
@@ -129,29 +82,18 @@ bool AacDecoderState::receive(uint8_t *data, DecoderSize *size) {
 
 AacDecoderState::AacDecoderState(uint32_t sample_rate, uint32_t channels) {
     constexpr size_t buffer_size = 8192;
-    io_buffer = std::shared_ptr<uint8_t>(reinterpret_cast<uint8_t *>(av_malloc(buffer_size)), &av_free);
-    io_context = std::shared_ptr<AVIOContext>(
-        avio_alloc_context(
-            io_buffer.get(), buffer_size, // buffer data
-            false, // writeable
-            this, // user data
-            [](void *user, uint8_t *data, int size) -> int {
-                auto *decoder = reinterpret_cast<AacDecoderState *>(user);
-
-                return decoder->read_packet_cache(data, size);
-            }, // read
-            nullptr,
-            nullptr),
-        &avio_close);
+    // io_buffer = std::shared_ptr<uint8_t>(reinterpret_cast<uint8_t *>(av_malloc(buffer_size)), &av_free);
```

Comment:
This two lines are unused. They can be deleted.

Reply:
Oki 👌

Reply:
done